protos/sigstore_verification.proto

```diff
@@ -162,3 +162,164 @@ message Input {
         // provided.
         optional Artifact artifact = 4;
 }
+
+// CertificateSummary contains verified metadata from the certificate
+message CertificateSummary {
+        // The issuer of the certificate
+        string certificate_issuer = 1;
+
+        // The Subject Alternative Name, or SAN, from the certificate
+        string subject_alternative_name = 2;
```

Comment:
This might not be an issue in practice, but flagging: X.509 technically has multiple SANs, and I don't think any of the Sigstore specifications/documentation explicitly say that Sigstore leaf certificates only have a single SAN. In practice this is true, but we _might_ want to flag some documentation improvements as a follow-up to this to emphasize that, as a matter of X.509 profile, Sigstore-compatible CAs only issue EE certs with singular SANs 🙂

Reply:
We probably should add the type for the SAN too?